packages/front-end/components/Experiment/SRMWarning.tsx

```diff
@@ -1,124 +1,237 @@
 import { FC, useState } from "react";
-import { formatTrafficSplit, getSRMNeededPrecisionP1 } from "@/services/utils";
+import { ExperimentReportVariation } from "back-end/types/report";
+import { FaExternalLinkAlt } from "react-icons/fa";
+import { MdInfoOutline } from "react-icons/md";
 import { useUser } from "@/services/UserContext";
 import { DEFAULT_SRM_THRESHOLD } from "@/pages/settings";
+import track from "@/services/track";
+import { pValueFormatter } from "@/services/experiments";
 import Modal from "../Modal";
+import Tooltip from "../Tooltip/Tooltip";
+import { ExperimentTab } from "./TabbedPage";
+import { useSnapshot } from "./SnapshotProvider";
+import VariationUsersTable from "./TabbedPage/VariationUsersTable";
+
+const NOT_ENOUGH_EVIDENCE_MESSAGE =
+  "There is not enough evidence to raise an issue. Any imbalances in the percentages you see may be due to change and aren’t cause for concern at this time.";
+
+const LearnMore = ({
+  type,
+  setOpen,
+  body,
+}: {
+  type: "simple" | "with_modal";
+  setOpen: (boolean) => void;
+  body: string | JSX.Element;
+}) => {
+  if (type === "with_modal") {
+    return (
+      <a
+        href="#"
+        onClick={(e) => {
+          e.preventDefault();
+          setOpen(true);
+        }}
+      >
+        Learn More {">"}
+      </a>
```

Comment:
Better to use button roles than an empty href
```suggestion
      <a
        role="button"
        className="a"
        onClick={() => setOpen(true)}
      >
        Learn More {">"}
      </a>
```